src/main/java/com/baller/presentation/controller/MemberController.java

```diff
@@ -0,0 +1,25 @@
+package com.baller.presentation.controller;
+
+import com.baller.application.service.MemberService;
+import com.baller.presentation.dto.request.member.SignUpRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/member")
```

Comment:
```suggestion
@RequestMapping("/api/members")
```
보통 REST API에서 리소스는 복수형으로 씁니다.